test/db-repo/change_history/entity/repository_test.go

```diff
@@ -0,0 +1,66 @@
+package change_history_test
```

Comment:
from the filename one could think this should test a repository, which it does not

Reply:
Well, this is a test implementation for a repository, but as long as you name it *_test.go, it can't be in the same package :|